### PR TITLE
Adjust random string generation to POSIX

### PR DESCRIPTION
**Pull request by dhaiducek**

### CODE REVIEW

**Comment by gurnben** [comment]
@dhaiducek Could you rebase this one onto master?

File: clusterclaims/apply.sh [hunk 1]
```diff
@@ -322,7 +322,10 @@ while [[ ("$CC_PEND_CONDITION" != "False" || "$CD_HIB_CONDITION" != "False" || "
     oc get clusterclaim ${CLUSTERCLAIM_NAME} -n ${CLUSTERPOOL_TARGET_NAMESPACE} -o json > $CC_JSON
     CC_NS=`jq -r '.spec.namespace' $CC_JSON`
     if [[ "$CC_NS" != "null" ]]; then
-        oc get clusterdeployment $CC_NS -n $CC_NS -o json > $CD_JSON
+        oc get clusterdeployment $CC_NS -n $CC_NS -o json > $CD_JSON 2> $CD_ERROR
```
Comment (dhaiducek) [inline clusterclaims/apply.sh]: Shoot. It doesn't like this:
```
./apply.sh: line 305: $CD_ERROR: ambiguous redirect
./apply.sh: line 325: $CD_ERROR: ambiguous redirect
```

Comment (dhaiducek) [reply clusterclaims/apply.sh]: I made a poor assumption that this was storing it to a variable. I've created an error file instead.

**Review by gurnben — APPROVED** [review]
Looks good to me - LGTM!

**Pull request merged**